cpu/sam3/periph/dac.c

```diff
@@ -64,8 +64,8 @@ void dac_poweron(dac_t line)
 {
     assert(line < DAC_NUMOF);
 
-    PMC->PMC_PCER1 = (1 << (ID_DACC - 32));
-    DACC->DACC_CHER = PMC_BIT;
+    DACC->DACC_CHER = (1 << line);
+    PMC->PMC_PCER1 = PMC_BIT;
```

Comment:
This seems incorrect to me: at least the order is broken now -> the module clock gate bit needs to be enabled before the actual module enable bit...

Reply:
right, will fix